backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/utils/CassandraAsyncExecutor.java

```diff
@@ -48,6 +48,12 @@ public CassandraAsyncExecutor(Session session) {
                 .publishOn(Schedulers.elastic()));
     }
 
+    public Mono<ResultSet> execute(String statement) {
```

Comment:
why can't you create a SimpleStatement outside the executor?

Reply:
fixed with: https://github.com/linagora/james-project/pull/3274

Reply:
sorry but you have to fix things in the right PR or it won't scale (I can't read/merge PRs fixup from different PRs in my head).

Reply:
ok i pushed a fixup similar to https://github.com/linagora/james-project/pull/3274